open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/EngineHostServicesConfig.java

```diff
@@ -75,8 +109,10 @@ public List<EngineServiceConfig> getEngineServiceConfigs()
 
     /**
      * Set up the list of configuration for the engine services that will run in this server.
+     * This configuration is only required if the services use a different metadata access store to
+     * the engine host.
```

Comment:
I wonder if it worth saying ,` because ...`

Reply:
I you want to change it, you can ...